pkg/js/parse_tests/029-dextendsub/düsseldorf.example.net.zone.ACTUAL

```diff
@@ -0,0 +1,9 @@
+$TTL 300
```

Comment:
The ".ACTUAL" files shouldn't be checked into git. They're artifacts left around to help people debug.

Reply:
Oh, yeah, sorry about that. Should we add them to `.gitignore` maybe?